src/Symfony/Bundle/FrameworkBundle/CacheWarmer/SerializerCacheWarmer.php

```diff
@@ -76,12 +76,12 @@ private function extractSupportedLoaders(array $loaders): array
 
         foreach ($loaders as $loader) {
             if ($loader instanceof XmlFileLoader || $loader instanceof YamlFileLoader) {
```

Comment:
Discussable change as it would be less efficient for "normal" loaders and the use of chain loaders is less common (?)
I like to get rid of potentially poorly performant code but in this example using array_merge in the loop might be better. What do you think?

Reply:
having chain loaders in this array is indeed much less common, as it would happen only if your runtime serializer has a LoaderChain nested inside the main LoaderChain instance, which is quite uncommon (FrameworkBundle itself will never produce such a configuration, so that will only happen if you write a compiler pass customizing the config)

Reply:
I'll remove the change in both CacheWarmers later today.